.changeset/thin-geckos-worry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/fiori-freestyle-writer': patch
+'@sap-ux/yaml': patch
```

Comment:
Shouldn't `ui5-config` also be patched (https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-config/package.json#L24)

Reply:
I think it will automatically be added. But I'll add it for clarity.